### PR TITLE
Move voltage scan range to NXelectronanalyser

### PR DESCRIPTION
**Pull request by lukaspie**

### CODE REVIEW

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 1]
```diff
@@ -185,6 +185,8 @@ NXmpes(NXobject):
         work_function(NX_FLOAT):
           unit: NX_ENERGY
           exists: recommended
+        voltage_energy_range(NX_FLOAT):
+          exists: optional
```
Comment (domna) [inline contributed_definitions/nyaml/NXmpes.yaml:189]: If it's optional we don't have to state it here explicitly (sorry, that slipped in the previous PR).

**Review by domna — APPROVED** [review]

File: contributed_definitions/nyaml/NXmpes.yaml (hunk 1 above)
Comment (lukaspie) [reply contributed_definitions/nyaml/NXmpes.yaml]: Right, we only make things optional if the term is not explictly named in the base class right, correct? So e.g., if only the generic group is defined. Like if there is an `(NXaperture)`, but we want to have the explicitly named (optional) `entrance_slit(NXaperture)` in `NXmpes`.

I removed it from NXmpes.

Comment (domna) [reply contributed_definitions/nyaml/NXmpes.yaml]: > Right, we only make things optional if the term is not explictly named in the base class right, correct?

Yes, exactly.

**Comment by lukaspie** [comment]
Can we still merge this PR before the workshop?

**Comment by domna** [comment]
> Can we still merge this PR before the workshop?

Let's just merge this now. I think it will be fine with @rettigl and we can discuss this at the workshop anyways.

**Pull request merged**